File: C/e84.c
#include <stdio.h>
#include "e84.h"
#include "exc.h"

int e84() {
    print("Enter the example you wnat to use from the selection below", 0, "s");
    print("Example1 - 1", 0, "s");
    print("Example2 - 2", 0, "s");
    int exi2;
    exi2 = input("i", 2);
    if (exi2 == 1) {
        example1d();
    } else if (exi2 == 2) {
        example2d();
    }
    return 0;
}

File: rust/src/e84.rs
pub mod example1;
pub mod example2;

pub fn main() {
    let mut exselected = String::new();
    println!("Enter the example you want to run from the following.");
    println!("Example1 - Enter 1 to run");
    println!("Example2 - Enter 2 to run");
    std::io::stdin().read_line(&mut exselected).unwrap();
    if exselected.trim() == "1" {
        example1::main();
    } else if exselected.trim() == "2" {
        example2::main();
    } else {
        println!("Invalid entry");
    }
}